### PR TITLE
[CHORE] "globalifies" the ReCaptcha SITEKEY for ease of change if needed later.

### PR DESCRIPTION
**Pull request by newb23**
# Description

"globalifies" the ReCAPTCHA sitekey for ease of change if needed later.

## Type of change

Please delete options that are not relevant.

- [x] Chore

# How Has This Been Tested?

Test Suites: 12 passed, 12 total
Tests:       103 passed, 103 total
Snapshots:   0 total
Time:        19.454 s
Ran all test suites.
Done in 20.35s.

Monday, May 9, 2022 22:11:46

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [ ] Screenshot if it includes any UI changes
- [x] I have read the contributing guidelines and agree to the T&Cs
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/lib/config.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ const TOKEN_CONTRACT = import.meta.env.VITE_TOKEN_CONTRACT;
 const DISCORD_REDIRECT = import.meta.env.VITE_DISCORD_REDIRECT;
 const CLIENT_VERSION = import.meta.env.VITE_CLIENT_VERSION as string;
 const RELEASE_VERSION = import.meta.env.VITE_RELEASE_VERSION as string;
+const SITEKEY = "6Lfqm6MeAAAAAFS5a0vwAfTGUwnlNoHziyIlOl1s";
```
Comment (spencerdezartsmith) [inline src/lib/config.ts:17]: It probably makes more sense to move this into the `.env.sample`. 🤔

**Review by spencerdezartsmith — CHANGES_REQUESTED** [review]
Great initiative! I think it makes more sense to move this key into the env file. Just left that comment.

**Comment by newb23** [comment]
@spencerdezartsmith 

I will move and test this just after the goblin modal PR +/- 10hrs from this comment.

**Comment by newb23** [comment]
Forgot to comment last night, but this is also completed. Moved the key into the .env.sample as requested!

**Review by Rofrtd — APPROVED** [review]
Hey @newb23 great improvement. 

Thanks for your contribution.
LGTM 🚀

**Pull request merged**